packages/@glimmer/babel-preset/index.js

```diff
@@ -75,9 +75,19 @@ module.exports = function (api, options) {
         { legacy: true },
       ],
 
-      __loadPlugins
-        ? require('@babel/plugin-proposal-class-properties')
-        : require.resolve('@babel/plugin-proposal-class-properties'),
+      [
+        __loadPlugins
+          ? require('@babel/plugin-proposal-class-properties')
+          : require.resolve('@babel/plugin-proposal-class-properties'),
+        { loose: true },
```

Comment:
I think we should make this configurable, and default it to `false`. Relying on loose mode semantics is not a good idea for future proofing, and this should generally be discouraged.